test/functional/mempool_limit.py

```diff
@@ -20,55 +23,76 @@ def set_test_params(self):
         ]]
         self.supports_cli = False
 
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
-
     def run_test(self):
         txouts = gen_return_txouts()
-        relayfee = self.nodes[0].getnetworkinfo()['relayfee']
+        # Helper function that generates large transactions
```

Comment:
you can remove this comment. the description of what `gen_return_txouts` does is best left for the doc string on that function itself